autogpt_platform/backend/backend/data/block.py

```diff
@@ -231,9 +269,32 @@ def __init__(
         self.contributors = contributors or set()
         self.disabled = disabled
         self.static_output = static_output
-        self.block_type = block_type
+        self.block_type = block_type if not webhook_config else BlockType.WEBHOOK
```

Comment:
Can we set this on the block level rather than making this assumption?

Reply:
shouldn't the block be setting that it is a webhook block?

Reply:
It can do that by specifying a webhook config